packages/prompts/test/spinner.test.ts

```diff
@@ -149,6 +149,31 @@ describe.each(['true', 'false'])('spinner (isCI = %s)', (isCI) => {
 		});
 	});
 
+	describe('indicator customization', () => {
+		test('custom frames', () => {
+			const result = prompts.spinner({ output, frames: ['🐴', '🦋', '🐙', '🐶'] });
+
+			result.start();
+
+			vi.advanceTimersByTime(80);
```

Comment:
if you do this in a loop, the snapshot will contain all 4 frames:

```ts
for (let i = 0; i < 4; i++) {
  vi.advanceTimersByTime(80);
}
```

or `80 * 4`. i think i prefer a loop to make it clear its for the 4 frames